src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -72,12 +89,13 @@ private void NodeAutoCompleteSearchControl_Unloaded(object sender, System.Compon
 
         private void CurrentApplicationDeactivated(object sender, EventArgs e)
         {
-            OnRequestShowNodeAutoCompleteSearch(ShowHideFlags.Hide);
+            OnRequestHideNodeAutoCompleteSearch();
```

Comment:
I think we don't want to hide the window on app deactivation anymore ; previously it was a popup and it was hiding by itself in many contexts.